test/addons-napi/test_general/test.js

```diff
@@ -87,8 +90,11 @@ test_general.testFinalizeWrap(z);
 test_general.removeWrap(z);
 z = null;
 global.gc();
-assert.strictEqual(test_general.finalizeWasCalled(), false,
-                   'finalize callback was not called upon garbage collection');
+const finalizeWasCalled = test_general.finalizeWasCalled();
+assert.strictEqual(finalizeWasCalled, false,
+                   'finalize callback was not called upon garbage collection' +
+                   'test_general.finalizeWasCalled() ' +
```

Comment:
This has awkward spacing (no space between line 1 & 2). Maybe better as:

> Finalize callback was not called upon garbage collection. test_general.finalizeWasCalled() returned ${finalizeWasCalled}

Reply:
@apapirovski 
Thanks for pointing this out. I added a space betwen the two lines